### ISSUE TITLE
Debian jessie/sid tw-cli seg faults

### ISSUE THREAD

**Issue by troex** [eLvErDe/hwraid#8]
Here is related KB from LSI site http://mycusthelp.info/LSI/_cs/AnswerDetail.aspx?inc=8374 with patched binary.

Looks like related issue with ubuntu http://serverfault.com/questions/525923/lsi-3ware-tw-cli-and-tdm2-segfault-with-debian-linux-kernels-after-3-8

I'm using successfully patched version with 3.14 kernel on Debian jessie

**Comment by joakimfors** [comment]
@troex Long shot, but do you happen to have the beta tw-cli binary for Jessie available? Can't find it on LSI^WAvago's site.

**Comment by troex** [comment]
Yeah looks like they messed up with drivers downloads and beta tools like tw_cli, luckly I still have the binary downloaded from LSI site which I can't find anymore on the web, I'll send you link via private message on github. This binary works for me Debian 8.0 Jessie 64bit on 3ware 9550SXU-12 controller.

**Comment by joakimfors** [comment]
Thanks!

**Comment by eLvErDe** [comment]
Hey,

The new binary is available from here:
http://www.avagotech.com/support/knowledgebase/1211161501805/debian-8-twcli-causes-fault-segment-failure

Can you confirm it works fine for you ? If so I'll try to get some time to update the package.

Thanks, Adam.
